test/browser/browser.sh

```diff
@@ -46,6 +46,17 @@ if grep -q 'ID=.*rhel' /etc/os-release; then
     dnf install -y kpatch kpatch-dnf
 fi
 
+# HACK: RHEL has these bundled in cockpit-system, but Fedora doesn't; Provides: break in CentOS/RHEL 10
+# due to https://issues.redhat.com/browse/TFT-2564
+if ! grep -q platform:el /etc/os-release; then
+    if [ "$PLAN" = "basic" ]; then
+        dnf install -y cockpit-kdump cockpit-networkmanager cockpit-sosreport
+    fi
+    if [ "$PLAN" = "network" ]; then
+        dnf install -y cockpit-networkmanager
+    fi
+fi
```

Comment:
This [log shows](https://artifacts.dev.testing-farm.io/65bede2d-b32c-4927-992c-810f7c3d6e73/) that we don't *strictly* need this -- the recommends of "cockpit" are enough. But we've been burned by relying on that before, so for now keep this. We can drop it again once EPEL 10 exists.